CRM/Contact/BAO/Contact/Utils.php

```diff
@@ -1034,17 +1034,12 @@ public static function updateGreeting($params) {
     }
 
     if (empty($filterContactFldIds)) {
-      $greetingDetails = [];
```

Comment:
if greetingDetails & filterContactFldIds are empty it is the same as returning as the following code is only invoked if they are not empty